src/main/java/lesson03/Action.java

```diff
@@ -0,0 +1,23 @@
+package lesson03;
+
+public enum Action {
+    CREATE,
+    SAVE,
+    LOAD,
+    EXIT,
+    ;
+
+    public static Action fromInteger(int value) {
```

Comment:
one entry - one exit. no multiple returns

Reply:
fixed, thank you